api_client/Cargo.toml

```diff
@@ -6,3 +6,4 @@ edition = "2021"
 
 [dependencies]
 vmm-sys-util = "0.12.1"
+thiserror = "1.0.52"
```

Comment:
Good - adding it the commit that uses it - but t comes before v in the alphabet so they should be the other way around.